gravatar/src/main/java/com/gravatar/services/GravatarResult.kt

```diff
@@ -3,15 +3,15 @@ package com.gravatar.services
 /**
  * Class representing the result of a network operation.
  */
-public sealed class Result<T, E> {
+public sealed class GravatarResult<T, E> {
     /**
      * Represents a successful fetch operation.
      *
      * @param T The type of the value fetched
      * @param E The error type
      * @property value The fetched value
      */
-    public data class Success<T, E>(public val value: T) : Result<T, E>()
+    public data class Success<T, E>(public val value: T) : GravatarResult<T, E>()
```

Comment:
Should we drop the `data` here?

Reply:
In this case, I think we can keep it, or do you see a risk in that?

Reply:
I don't think we would add more properties to it, so I do also think it's safe.